ibazel/ibazel.go

```diff
@@ -372,3 +429,15 @@ func (i *IBazel) watchFiles(query string, watcher *fsnotify.Watcher) {
 
 	fmt.Fprintf(os.Stderr, "Watching: %d files\n", successFullWatchFileCount)
 }
+
+func testPort(port uint16) bool {
```

Comment:
This is nitpicky, but can you fix the indentations in this function?

Reply:
Done. Changed to tabs.

Reply:
Can you also run gofmt?